src/e2e/java/teammates/e2e/util/ResultsStatistics.java

```diff
@@ -0,0 +1,75 @@
+package teammates.e2e.util;
+
+/**
+ * Result statistics class for L&P tests.
+ */
+public class ResultsStatistics {
+
+    private double meanResTime;
+    private double pct1ResTime; // 90th percentile
+    private double throughput;
+    private int errorCount;
+    private int sampleCount;
+    private String resultsFeedback = "";
+    private String testName;
+
+    /**
+     * Generate a feedback message based on results statistics with the provided threshold.
+     */
+    public void generateResultsFeedback(double errorRateLimit, double meanResTimeLimit) {
+        checkErrorLimit(errorRateLimit);
+        checkMeanResTimeLimit(meanResTimeLimit);
+        if ("".equals(resultsFeedback)) {
+            System.out.println("You have successfully passed the default profiling threshold for " + testName);
+        } else {
+            throw new AssertionError(resultsFeedback);
+        }
+    }
+
+    /**
+     * Generate the statistics from a given the test result.
+     */
+    public void generateResultsStatistics() {
```

Comment:
I think ```Report.log()``` cannot print to the console, it seems to be generated to the HTML file.

![image](https://user-images.githubusercontent.com/28522090/56472606-05b18180-6493-11e9-842d-4fd200654871.png)

Also, should I just throw an assert error and there is no need to show the success msg?

Reply:
> I think Report.log() cannot print to the console, it seems to be generated to the HTML file.

Oh, right. Should be `Reporter.log(printString, true)`

Reply:
> Also, should I just throw an assert error and there is no need to show the success msg?

Yep, as discussed in the [comment earlier](https://github.com/TEAMMATES/teammates/pull/9652#discussion_r277137671), the `"You have successfully..."` is not needed.

Reply:
Noted on that! Should I replace the `AssertionError` with `Report.log()` instead? At least it wont cause any test to fail?

Reply:
The reason we want `AssertionError` is so that it is marked with a red `FAILED` in the build output.
Also, `generateResultsFeedback()` (which contains the assertion) is only called after `generateResultsStatistics()` (which displays the stats), right? So, it shouldn't fail early